### ISSUE TITLE
Feature request: allow lists for active_dims in RBFKernel()

### ISSUE THREAD

**Issue by eytan** [cornellius-gp/gpytorch#302]
When specifying the `active_dims`, one is able to use a list when on a CPU, but on a GPU, one needs to manually construct pytorch tensors for the appropriate device.  This results in some pretty ugly code to construct fully additive kernels, like:

```python
        def cudafy(a_list):
            if cuda:
                return torch.tensor(a_list, dtype=torch.long, device="cuda")
            else:
                return a_list
         self.covar_module = RBFKernel(active_dims=cudafy([0]))
         for i in range(1, latent_dimensions):
              self.covar_module += RBFKernel(active_dims=cudafy([i]))
```

It would be a lot more convenient if one didn't have to worry about whether one were in GPU/CPU mode, so that one could just write:
```python
         self.covar_module = RBFKernel(active_dims=[0])
         for i in range(1, latent_dimensions):
              self.covar_module += RBFKernel(active_dims=[i])
```

**Comment by Balandat** [comment]
In your last example the first `RBFKernel` doesn't know the device unless we tell it what it should live on. Adding that into the constructor would be pretty ugly tough.

What we could do instead is make the active_dims a buffer then calling `.cuda()` on the overall model will do the right thing. We could also add logic that automatically moves components of an additive kernel to the GPU if the first of the terms lives there, but that might be overkill.

I can put up a PR for this.

**Comment by Balandat** [comment]
@eytan, with #303 you can just add create the model as usual, and then call `.cuda()` on the overall model, this will automatically cudaify the `active_dims` in all attached kernels.

**Comment by gpleiss** [comment]
Closed by #303
